common_logging/src/test/java/uk/gov/dhsc/htbhf/logging/EventLoggerSpringConfigTest.java

```diff
@@ -57,4 +61,21 @@ void shouldLogMetadataInOrderGivenDataNotInOrder() {
         assertCorrectLoggingMessage(EVENT_AS_STRING_WITH_ORDERED_METADATA, INFO);
     }
 
+    @Test
+    void shouldLogEventSubclass() {
+        //Given
+        ApplicationStartedEvent applicationStartedEvent = ApplicationStartedEvent.builder()
+                .applicationId("myApp")
+                .applicationVersion("1.0.1")
+                .instanceIndex("1")
+                .build();
+        //When
+        eventLogger.logEvent(applicationStartedEvent);
+
+        //Then
+        String message = getSingleMessageContent(INFO);
```

Comment:
As per my comment above, I think that the unit tests in this class should be doing assertions in the same way, can you try to combine the two so that they are all asserting in the same way? Ok, in the tests above we are controlling the timestamp, but that aside, we should be doing assertions in the same way for each test as tey're essentially the same assertions.

Reply:
Would be good to still use common assertions in this unit test class.

Reply:
How about `assertSingleLogMessageHasText` and `assertSingleLogMessageContainsText`? Otherwise you might have to suggest how a single assertion would work.

Reply:
That sounds a pretty good suggestion. To be honest I hadn't looked into it too deeply, it just seemed like they were doing very similar things in different ways so thought there would be a common ground they could share.